openapi-build-extension/src/main/java/io/ballerina/openapi/build/HttpServiceAnalysisTask.java

```diff
@@ -68,6 +69,11 @@
  * @since 2.0.0
  */
 public class HttpServiceAnalysisTask implements AnalysisTask<SyntaxNodeAnalysisContext> {
+    static boolean isErrorPrinted = false;
```

Comment:
Whats the purpose of having this flag? can the analysis get triggered multiple times during a single compilation?